src/models/patient.model.ts

```diff
@@ -0,0 +1,50 @@
+import dayjs from "dayjs";
+export default class Patient {
```

Comment:
It's uncommon to have a model like this in frontend code. I don't like it. Instead, you just use the api object itself as an object, with no need for wrapper objects.

Reply:
The reason I chose to use it is the api object itself can be very nested and complicated. The wrapper offers a simple and typed way of accessing the object and can offer centralized error handling and type casting for certain properties. Is there another way to achieve the same without using the object directly?

Reply:
The more react way is to embed things into your jsx:

```js
function PatientSearchResults(props) {
  return props.patients.map(renderPatient)

  function renderPatient(patient) {
    const preferredIdentifier = patient.identifiers.find(i => i.preferred) || patient.identifiers[0]
    
    return (
      <div key={patient.uuid}>
        <div>{patient.uuid}</div>
        <div>{patient.person.display}</div>
        <div>
          {patient.identifiers.map(id => (
            <div>{id.identifier} {id.identifierType.display}</div>
          ))}
        </div>
        <div>{dayjs(patient.birthdate).format("DD-MMM-YYYY")}</div>
        <div>{patient.person.age}</div>
        <div>{patient.person.gender}</div>
      </div>
    )
  }
}
```

Reply:
@joeldenning  I updated this PR and also added tests.